doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -66,12 +70,15 @@ The FIWARE NGSI (Next Generation Service Interface) API defines
 
 ### Context data modelling and exchange
 
+![NGSI data model](https://raw.githubusercontent.com/telefonicaid/fiware-orion/develop/doc/apiary/v2/Ngsi-data-model.png)
+
 #### Context Entities
 
 Context entities, or simply called entities, are the center of gravity
 in the FIWARE NGSI information model. An entity represents a
-real-world object like e.g. a sensor, a person, or a room. Each entity
-is uniquely identified by its **entity id**. 
+real-world or logical object like e.g. a sensor, a person, a room, or an
```

Comment:
An entity represents a thing i.e any physical or logical object in the real world (eg. a sensor, a person, room, an issue, ...)